File: tools/findIntervalSum.py
def findIntervalSum (a, b):
    if a < b:
        minimum = a
        maximum = b
        suma = 0
        for i in range(minimum, maximum + 1, 1):
            suma = suma + i
        return suma
    else:
        minimum = b
        maximum = a
        suma = 0
        for i in range(minimum, maximum + 1, 1):
            suma = suma + i
        return suma

print(findIntervalSum(12,15))
print(findIntervalSum(1,3))

File: tools/findIntervalSum.js
function findIntervalSum (a, b) {
    var minimum = a;
    var maximum = b;
    var suma = 0;
    if (a < b) {
        for (let i = minimum; i < b + 1; i = i + 1) {
            suma = suma + i;
        }
        return suma;
    } else {
        for (let i = minimum; i < b + 1; i = i + 1) {
            suma = suma + i;
        }
        return suma;
    }
}

console.log(findIntervalSum(12,15))
console.log(findIntervalSum(1,3))